packages/modules/order/src/migrations/Migration20240827133639.ts

```diff
@@ -3,7 +3,15 @@ import { Migration } from "@mikro-orm/migrations"
 export class Migration20240827133639 extends Migration {
   async up(): Promise<void> {
     this.addSql(
-      'alter table if exists "return_item" add column if not exists "damaged_quantity" numeric not null default 0, add column if not exists "raw_damaged_quantity" jsonb not null;'
+      'alter table if exists "return_item" add column if not exists "damaged_quantity" numeric not null default 0, add column if not exists "raw_damaged_quantity" jsonb;'
+    )
+
+    this.addSql(
+      `UPDATE "return_item" SET raw_damaged_quantity = '{"value": "0", "precision": 20}'::jsonb;`
```

Comment:
q: do we want to set this as a default value? or let the app code handle it?

Reply:
Hmm, my immediate thought was that this should not be part of the migration. How have we dealt with this elsewhere?

Reply:
I don't think we've had to run null migrations on big number js fields with data before.

Reply:
I would say we let the app manages it, I don't think we have any default for other big numbers